test/printer/samples/element-with-class-and-comment-inside.html

```diff
@@ -1,5 +1,3 @@
 <div class="test">
     <!-- c -->
 </div>
-
```

Comment:
Any reason why you did remove this test? IMO it's not equivalent since it is a comment without whitespace before/after.

Reply:
It now formats to the same thing. The only reason it didn’t before is that the first comment was interpreted as a “don’t format the next line”